DataLayer/Migrations/20190414094415_AddedApplicationUserAndFriendship.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.IO;
+using GameBoard.DataLayer.Migrations.MigrationBuilderExtensions;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GameBoard.DataLayer.Migrations
+{
+    public partial class AddedApplicationUserAndFriendship : Migration
```

Comment:
A reminder to regenereate the migration from the model after changes in `DbContext.Model`.

Reply:
Did it.